### ISSUE TITLE
robot protocol ev3dev - connection issues

### ISSUE THREAD

**Issue by rbudde** [OpenRoberta/openroberta-lab#248]
- after connecting the ev3dev everything is fine
- if a press "disconnect" and then "connect" immediately after each other, a new token pops up and vanishes immediately
- the new token hits the server, but cannot be entered by the user (as he/she can't read it)

**Comment by ensonic** [comment]
Yes, unfortunately. I cannot cancel the pending connection. If you disconnect, you need to wait right now until it is connected. I'll take another look to see if there is a better way to do this.
